src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -11,12 +11,12 @@ export class LocalTaskService implements TaskService {
   private static readonly STORAGE_KEY: string = 'tiny.tasks';
 
   getAll(): Observable<Task[]> {
-    return of(this.readTasks());
+    return of(this.readTasks().sort((x,y) => x.completed === y.completed ? 0 : x? 1 : -1));
```

Comment:
This does not work. 😕  As you've correctly seen one of the acceptance criteria is to sort the tasks by done, to the bottom. The express is almost complete but make it work but one crucial detail is missing. For the second ternary operator you are missing the property you want to sort against.

`x.completed `

making this work with

`return of(this.readTasks().sort((x,y) => x.completed === y.completed ? 0 : x.completed ? 1 : -1));`

I guess think went throught because there was no test case to make sure the sorting works as expected.

Reply:
You are right, totally missed it here!